packages/bot-web-ui/src/components/quick-strategy.jsx

```diff
@@ -462,6 +553,16 @@ const QuickStrategy = props => {
                         <MobileWrapper>
                             <Div100vhContainer className='quick-strategy__wrapper--is-mobile' height_offset='80px'>
                                 <ContentRenderer {...props} />
+                                <Modal
+                                    className='modal--info'
+                                    title={info_label}
+                                    is_open={is_info_modal_open}
+                                    toggleModal={toggleInfoModal}
+                                >
+                                    <Modal.Body>
+                                        <div className='quick-strategy_info-message'>{info_message}</div>
```

Comment:
I thinks it's better if we localize message and label here instead of localize messages inside icons

Reply:
Is it not better for translations to localize each string and not localize variables?

Reply:
I really don't know :joy: